MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -429,12 +431,22 @@ public Task<MediaInfo> GetMediaInfo(MediaInfoRequest request, CancellationToken
                 analyzeDuration = "-analyzeduration " + ffmpegAnalyzeDuration;
             }
 
+            if (analyzeDuration.Length > 0)
```

Comment:
check for empty or set analyzeDuration to null

Reply:
I will use the string.IsNullOrEmpty as above. Not sure why this changes anything.

Reply:
code style